backend/src/main.rs

```diff
@@ -24,14 +33,44 @@ async fn main() {
     }
 }
 
+pub struct RunContextInner {
+}
+impl RunContextInner {
+}
```

Comment:
ini kenapa kosong?

Reply:
Placeholder buat lifetime RunContext, also buat `mpsc` (still in doubt)